vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow/src/main/java/com/vaadin/flow/component/confirmdialog/ConfirmDialog.java

```diff
@@ -695,6 +696,174 @@ public void setCloseOnEsc(boolean closeOnEsc) {
         getElement().setProperty("noCloseOnEsc", !closeOnEsc);
     }
 
+    /**
+     * Confirm dialog does not support adding content.
```

Comment:
Let's mention that `setText` should be used instead to modify the content of the dialog. Same for the other methods that modify the contents.

Reply:
Added mentions of `setText` to initialize content.